website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -64,32 +99,56 @@ This means you can automate deployment of the gateway using your automation
 method of choice, adding as many gateways as necessary for failover and load
 balancing.
 
-<Alert
-  color="warning"
-  html={`
-    <strong>Note:</strong> Be sure to set a unique <code>FIREZONE_ID</code> for each
-    gateway you deploy. This can be any non-empty string and is used to
-    identify the gateway in the portal for audit trail and logging purposes.
-  `}
-/>
+<Alert color="warning">
+  **Note:** Be sure to set a unique `FIREZONE_ID` for each gateway you deploy.
+  This can be any non-empty string and is used to identify the gateway in the
+  portal for audit trail and logging purposes.
+</Alert>
+
+## High availability
+
+Firezone was designed from the ground up to support high availability
+requirements. This is achieved through a combination of **automatic failover**
+and **load balancing**, both of which are automatically enabled on all plans.
+
+### Automatic failover
+
+Two or more gateways can be deployed within a site to achieve automatic failover
+for client connections to resources in that site.
+
+Here's how it works:
+
+- When the admin portal detects a particular gateway is unhealthy, it will stop
+  using it for new connection requests to resources in the site.
+- Existing clients will remain connected to the gateway until they themselves
+  detect it to be unhealthy.
+- Clients detect unhealthy gateways by maintaining keepalive timers with
+  gateways they're connected to. If the keepalive timer expires, the client will
+  disconnect from that gateway and ask the portal for a new, healthy one to
+  connect to.
+- On average, the client keepalive timer expires after **20 seconds**. This is
```

Comment:
Hope we will be able to tune it down to 2-5 in the future, being on a video call on the go is not a joke anymore with Vision Pro ppl :).